src/main/resources/openapiv2.json

```diff
@@ -219,6 +219,39 @@
       ]
     },
     "/consumers/{groupid}/instances/{name}/subscription": {
+      "get": {
+        "consumes": [
```

Comment:
It's a GET, it doesn't consume any data. The client doesn't send any JSON in the request.